linera-chain/src/manager/single.rs

```diff
@@ -99,6 +99,9 @@ impl SingleOwnerManager {
     }
 
     pub fn verify_owner(&self, proposal: &BlockProposal) -> Option<PublicKey> {
+        if proposal.content.round != RoundNumber::ZERO {
```

Comment:
Fyi we have this test above as well. In both cases, it's a little buried in the code and not failing explicitly.

Reply:
Ah, so this is redundant?
Happy to remove it. But it's cheap, and since the function is public maybe it makes sense here, too.

Reply:
We already verify the round in `check_proposed_block` (as we should) and `create_vote` (just in case?). So yes I would remove this one.